worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/blocks/CharBlocks.java

```diff
@@ -18,12 +17,11 @@ public abstract class CharBlocks implements IBlocks {
 
     protected static final Section FULL = new Section() {
         @Override
-        public char[] get(CharBlocks blocks, int layer) {
-            char[] arr = blocks.blocks[layer];
+        public char[] get(CharBlocks blocks, int layer, char[] arr) {
             if (arr == null) {
```

Comment:
in FULL, the array is never null if we always call it with a null check for that array directly before. I guess the warnings could be removed?

Reply:
Yeah tbh I just did only the changes required to test that stuff didn't implode